### DISCUSSION TITLE
How do you rebuild the lock file (without deleting it)?

### DISCUSSION THREAD

**Discussion by matthewfeickert** [prefix-dev/pixi#1070] [Q&A] [answered]
While rebuilding the lock file for `pixi` [`v0.17.1`](https://github.com/prefix-dev/pixi/releases/tag/v0.17.1) with

```
rm pixi.lock && pixi install
```

is gloriously fast ( :rocket: ) it would be good to have some instructions in the [`pixi install` docs](https://pixi.sh/v0.17.1/cli/#install) about how to rebuild the lock file from scratch if desired.

From Issue https://github.com/prefix-dev/pixi/issues/73, it seems like `pixi update` might be the way to do this in the future, but is there a way to do anything from a single CLI API now (in `v0.17.1`)?

**Comment by ruben-arts** [comment, marked as answer]
He @matthewfeickert, 
Without deleting it is not possible to do a complete re-render. I added a short explainer on your request: https://github.com/prefix-dev/pixi/pull/1073

`pixi updated` really shouldn't be far out now at the moment of writing this!

**Reply by matthewfeickert** [reply]
Thanks very much for the swift answer, @ruben-arts!

> `pixi update` really shouldn't be far out now at the moment of writing this!

That's exciting news. :)

**Comment by pavelzw** [comment]
I created a diff tool and added some documentation on how to do this in GitHub Actions, see https://github.com/prefix-dev/pixi/pull/1456
